ssp

```diff
@@ -1624,6 +1628,16 @@ sub check_for_os_release_32bit {
     print_critical('         \_ Send customer this premade: "MIGRATION - 32-bit CentOS/RHEL/CL EOL"');
 }
 
+sub check_for_ea3 {
+    return if $EA4;
```

Comment:
if $EA4 is already defined we wouldn't want to even call this sub.  The call for this check should never even take place if we already know it's an EA4-based system.

Reply:
That's the entire purpose of returning, this sub isn't relevant for EA4, but it makes more sense to check here than before the sub is called because that's how SSP is structured.  Maybe not ideal, but it's consistent.